src/unlink/unlink.rs

```diff
@@ -68,7 +68,8 @@ pub fn uumain(args: Vec<String>) -> i32 {
         );
     }
 
-    let c_string = CString::new(matches.free[0].clone()).unwrap(); // unwrap() cannot fail, the string comes from argv so it cannot contain a \0.
+    let c_string = CString::new(matches.free[0].clone()).unwrap();
+    //^ unwrap() cannot fail, the string comes from argv so it cannot contain a \0.
```

Comment:
Move this to the line before `let c_string = ...`.

Reply:
Done.